packages/radio/src/radio.css

```diff
@@ -23,7 +23,7 @@ governing permissions and limitations under the License.
 /**
  * Work around for https://github.com/adobe/spectrum-css/issues/2251
  **/
-:host([dir="rtl"]) #button:after {
+:host([dir='rtl']) #button:after {
```

Comment:
We need to hunt down why we're getting spurious linting included in unrelated PRs.

Reply:
I dont think this came from linting. Must be coz of yarn install

Reply:
`singleQuote: true` in `.prettierrc.yaml` is the culprit.